Refine translations section of CONTRIBUTING.md (#1449)
loot/loot: CONTRIBUTING.md

To translate everything but masterlist messages, first fork this repository. All file paths given below are relative to its base folder.

Keep in mind that you don't need to submit a perfect pull request; just follow the instructions below to the best of your ability and we will correct any coding-related mistakes when we review your submission.

### Translating the Installer

First check that an [Inno Setup translation](http://www.jrsoftware.org/files/istrans/) exists for your language. Unofficial translations are acceptable, but require a bit of extra handling. If there isn't an official or unofficial translation for Inno Setup, you're better off making a translation and getting it listed on the linked page before continuing.

1. Open the installer script at `scripts/installer.iss` in a text editor of your choice.
2. If your language only has an unofficial translation, add a `#define <Language>Exists` block for it near the top of the script, like it has been done for Korean and Simplified Chinese.

> 3. Add your language to the `[Languages]` section. The `Name` must be the [POSIX locale code](https://www.gnu.org/software/gettext/manual/html_node/Locale-Names.html) for your language. The `MessagesFile` filename is the filename of the Inno Setup translation that you checked exists. If your language only has an unofficial translation, wrap its line in `#ifdef` and `#endif` lines, again like it has been done for Korean and Simplified Chinese.

Reviewer: ```suggestion
3. Add your language to the `[Languages]` section. The `Name` should be the [POSIX locale code](https://www.gnu.org/software/gettext/manual/html_node/Locale-Names.html) for your language. The `MessagesFile` filename is the filename of the Inno Setup translation that you checked exists. If your language only has an unofficial translation, wrap its line in `#ifdef` and `#endif` lines, again like it has been done for Korean and Simplified Chinese.
```

In the `[Languages]` section the `Name` value can be anything, it's just got to match the relevant entry in `[CustomMessages]`. The locale code is an obvious choice though.

### Adding A New Translation

> If you're adding a new translation, LOOT's source code must be updated to recognise it. While you're welcome to do this yourself and include the changes in your translation's pull request, it's not necessary. You should focus on the translation and following the instructions above; we can handle the rest. The files and functions which must be updated are given below.

Reviewer: I think "you should" here is a bit strong, I'd say something like "If you're not comfortable making these changes, you can leave them out of your pull request.".

Author: That's a fair point, how about this:

```suggestion
If you're adding a new translation, LOOT's source code must be updated to recognise it. While you're welcome to do this yourself and include the changes in your translation's pull request, you don't have to if you're not comfortable with it; we can add it later. The files and functions which must be updated are given below:
```

Also, I just noticed, but the period at the end should probably be a colon since it leads into a list.
